### DISCUSSION TITLE
what can I do to change the head in file "bin/airflow" to "#!/usr/bin/env"

### DISCUSSION THREAD

**Discussion by mingkeming** [apache/airflow#14948] [Q&A]
when I run pip install --target= path apache_airflow-2.0.1-py3-none-any.whl
I get all the package ,and I found some new file were created  in 
![image](https://user-images.githubusercontent.com/19298750/112110147-76b4c300-8bed-11eb-8cce-3504f575563b.png)

then when I copy these package to the other platform , some error occured,
all reason is the file "airflow" in bin has the head 
![image](https://user-images.githubusercontent.com/19298750/112109089-14a78e00-8bec-11eb-8574-e52bfdcba20b.png)
is someone know , what can I do to change "#!/opt/buildtools/python-3.8.5/bin/python3.8" to "#!/usr/bin/env"

when I use it like[ python -m pip install --target=path airflow.whl],the head has changed to other .
is this need to be a issure  for pip?
who can tell me ? thanks

**Comment by turbaszek** [comment]
Any particular reason to install form wheel? Why not use `pip install apache-airflow`?

**Reply by mingkeming** [reply]
because the pc A and B  is offline。 we must install it in one pc A，and then copy all files to the other pc B。

**Comment by potiuk** [comment]
How do you build your wheels? Are you using Airflow ones or you build them yourself? I believe you are building it yourself. Distutlil has a bug apparently that does this shebang modification if you have "unclean" virtualenv and you run the build in not the right sequence.

Just follow this comment and discussion to fix it https://github.com/pybuilder/pybuilder/issues/168#issuecomment-156400193
